nodejs-env/src/main/scala/org/scalajs/jsenv/nodejs/NodeJSEnv.scala

```diff
@@ -164,6 +165,37 @@ object NodeJSEnv {
         case Input.CommonJSModulesToLoad(modules) =>
           for (module <- modules)
             writeRequire(module)
+
+        case Input.ESModulesToLoad(modules) =>
+          if (modules.nonEmpty) {
+            val uris = modules.map {
+              case module: FileVirtualBinaryFile =>
+                module.file.toURI
+              case module =>
+                tmpFile(module.path, module.inputStream).toURI
+            }
+
+            val imports = uris.map { uri =>
+              s"""import("${escapeJS(uri.toASCIIString)}")"""
+            }
+            val importChain = imports.reduceLeft { (prev, imprt) =>
+              s"""$prev.then(_ => $imprt)"""
+            }
+
+            val importerFileContent = {
+              s"""
+                |$importChain.catch(e => {
+                |  /* Make sure to fail the process, but give time to Node.js to
+                |   * display a good stack trace before that.
+                |   */
+                |  setTimeout(() => process.exit(1), 100);
+                |  throw e;
```

Comment:
You can simply `console.log(e)` and then exit. See [this answer](https://stackoverflow.com/a/33593443/1149944)